extensions/wikia/HTTPSSupport/tests/HTTPSSupportHooksTest.php

```diff
@@ -86,4 +86,78 @@ public function onBeforeResourceLoaderCSSMinifierDataProvider() {
 			]
 		];
 	}
+
+	public function testOnLinkerMakeExternalLink() {
+
+		$exampleString = '';
+		$exampleBool = false;
+		$exampleArray = [];
+
+		#Should not be upgraded to HTTPS
+
+		#external URL
+		$url = 'http://www.example.com';
```

Comment:
we should also make sure `https://www.example.com` is not downgraded to `http://www.example.com`